app/router.js

```diff
@@ -33,12 +33,21 @@ app.get('/_s/:user', (req, res) => {
   log.debug('user promises', req.params.user)
 
   req.user.getPromises({
+    where: {
+      void: {
+        [Sequelize.Op.not]: true
+      }
+    },
     include: [{
       model: Users
     }],
     order: [['tfin', 'DESC']],
   }).then(promises => {
-    const reliability = _.meanBy(promises, 'credit')
+    const reliability = _(promises)
```

Comment:
this is a cleaner/more efficient implementation of https://github.com/beeminder/iwill/pull/131/commits/dc9bdc804866bac61e7e4628a27f192e718c62a0#diff-f3a289058604b2b069d07bb8e2cda60cR43